src/app/(proper_react)/redesign/(authenticated)/user/welcome/details/page.tsx

```diff
@@ -0,0 +1,92 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { isEligible } from "../../../../../../functions/server/onerep";
+import Script from "next/script";
+
+export function generateMetadata() {
+  return {
+    title: "Welcome - Details",
+  };
+}
+
+export default async function UserWelcomeDetails() {
+  const eligible = await isEligible();
+
+  if (!eligible) {
+    return (
+      <main>
+        <h2>You have already used your free scan.</h2>
+      </main>
+    );
+  } else {
+    return (
+      <>
+        <Script
+          type="module"
+          src="/nextjs_migration/client/js/welcome-details.js"
+        ></Script>
+
+        <main>
+          <h1>Enter the details you want to protect</h1>
+          <p>
+            We’ll use this to find exposures of your personal information, and
+            then guide you step-by-step on how to fix it.
+          </p>
+
+          <section>
+            <form name="details">
+              <table>
+                <tbody>
+                  <tr>
+                    <td>
+                      <label htmlFor="firstname">First name: </label>
+                    </td>
+                    <td>
+                      <input type="text" name="firstname" id="firstname" />
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>
+                      <label htmlFor="lastname">Last name: </label>
+                    </td>
+                    <td>
+                      <input type="text" name="lastname" id="lastname" />
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>
+                      <label htmlFor="citystate">City and state: </label>
+                    </td>
+                    <td>
+                      <input type="text" name="city" id="city" />
+                    </td>
+                    <td>
+                      <input type="text" name="state" id="state" />
+                    </td>
```

Comment:
Yes, I think the JSON route that parses this is the more important bit (I didn't want to deal with parsing the city/state here since that's being worked on in parallel as part of a lookup component